examples/Expat/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2014, Ruslan Baratov
+# Copyright (c) 2015, Alexander Lamaison
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(example-hunter-expat)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+hunter_add_package(EXPAT)
```

Comment:
I think it must be 'hunter_add_package(Expat)' here. Probably on OS like Mac or Windows it doesn't matter but on case sensitive platforms like Linux it will be an error.